tutorials/snap/setup-ubuntu-core-joule.md

```diff
@@ -0,0 +1,227 @@
+categories: snap
+status: Published
+feedback-link https://github.com/ubuntu/codelabs/issues
+tags: snap, usage, beginner, joule, console-conf
+difficulty: 1
+published: 2016-12-10
+Author: Didier Roche <didier.roche@canonical.com>
+
+---
+
+# Setup Intel Joule
+
+## Introduction
+Duration: 15:00
+
+Welcome to the world of Ubuntu Core! In this codelab, we are going to introduce you how to flash your eMMC for your Intel Joule board, and finally turn your Intel Joule into a Ubuntu Core 16 system.
+
+![IMAGE](https://assets.ubuntu.com/v1/3647779b-joule-product.png)
+
+
+### What you’ll learn
+  - Create a Ubuntu SSO (Single-Sign-On) account
+  - How to setup your hardware environment
+  - How to flash Ubuntu Core 16 for your Intel Joule board
+  - How to do console-conf for your Ubuntu Core 16 system
+### What you’ll need
+  - 1 Intel Joule Dev Kit
+  - 1 HDMI monitor
+  - 1 USB keyboard
+  - 1 USB mouse
+  - 1 USB hub (connecting both keyboard and mouse)
+  - 1 micro HDMI to HDMI cable (connecting HDMI monitor)
+  - 1 SD card adapter
+  - 1 Ubuntu Desktop (16.04) or Windows system
+
+Survey
+: How will you use this tutorial?
+ - Only read through it
+ - Read it and complete the exercises
+: What is your current level of experience?
+ - Novice
+ - Intermediate
+ - Proficient
+
+## Getting set up
+Duration: 7:00
+
+### Register a Ubuntu SSO account
+
+If you have already had an SSO account, you can skip this step and move to the next. The account is used for console-conf in the following steps.
+
+  - Ubuntu SSO account is required to setup user profile. A user needs to create a ubuntu SSO account before using Ubuntu Core 16
+  - To create Ubuntu SSO account, go to https://login.ubuntu.com
+
+![IMAGE](https://assets.ubuntu.com/v1/af05aa4e-sso-1.png)
+
+  - Complete account verification
+
+![IMAGE](https://assets.ubuntu.com/v1/11e60c02-sso-2.png) 
+
+![IMAGE](https://assets.ubuntu.com/v1/fcaa5d3d-sso-3.png) 
+
+![IMAGE](https://assets.ubuntu.com/v1/bec1518f-sso4.png) 
+
+  - Create a SSH Key
+	This step is only necessary if you don’t have a SSH key on your system.
+  - On your Ubuntu desktop, here is how you would create it:
+
+
+```bash
+$ mkdir ~/.ssh
+$ chmod 700 ~/.ssh
+$ ssh-keygen -t rsa
+```
+
+You will be prompted for a location to save the keys, and a passphrase for the keys. This passphrase will protect your private key while it's stored on the hard drive:
+
+
+```bash
+Generating public/private rsa key pair.
+Enter file in which to save the key (/home/b/.ssh/id_rsa):
+Enter passphrase (empty for no passphrase):
+Enter same passphrase again:
+Your identification has been saved in /home/b/.ssh/id_rsa.
+Your public key has been saved in /home/b/.ssh/id_rsa.pub.
+```
+
+Your public key is now available as `.ssh/id_rsa.pub` in your home folder.
+
+  - For Window users, please refer to the following link about how to generate SSH key: [https://support.rackspace.com/how-to/generating-rsa-keys-with-ssh-puttygen].
+
+  - Import SSH Key
```

Comment:
Yes, Markdown is having a hard time with some of these lists because of all the paragraph breaks and images. I will just hard code the numbers (I dropped number 2 because 1+2 as separate steps was silly.